api/src/main/java/marquez/db/RunDao.java

```diff
@@ -151,7 +151,7 @@ default void updateJobVersionUuid(UUID rowUuid, Instant updatedAt, UUID jobVersi
       SELECT_RUN
           + "INNER JOIN job_versions AS jv ON r.job_version_uuid = jv.uuid "
           + "WHERE jv.namespace_name = :namespace and jv.job_name = :jobName "
-          + "ORDER BY r.created_at DESC "
+          + "ORDER BY STARTED_AT DESC NULLS LAST "
```

Comment:
I think this should be:

```
ORDER BY r.started_at DESC NULLS LAST
```

Reply:
The runs table doesn't have a `started_at` time. This is using the alias from the `SELECT_RUN` constant which is the joined in `transitioned_at` column from `run_states`.

Reply:
Ahh, I see 👍